### PR TITLE
added static config parse process and config file example

### PR DESCRIPTION
**Pull request by dolow**
This PR adds config file option to `sgmed`.
It allows to manage options with static config file.

- [x] old way exporting (no regression)
- [x] export with config file

### CODE REVIEW

File: packages/scene-graph-mediator/client/example/sgmed.config.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module.exports = () => {
+  return {
+    runtime: 'cc2',
+    assetRoot: 'CocosCreatorProject/assets',
+    sceneFiles: [
+      'CocosCreatorProject/assets/Scene/sample.fire'
+    ],
+    plugins: [
+      // define module name or path
+      '/Users/kuwabara_yuki/workspace/git/scene-graph/packages/scene-graph-event/client/lib/exporter/CocosCreator',
```
Comment (dolow) [inline packages/scene-graph-mediator/client/example/sgmed.config.js]: it should be changed to object obtained via `require()`;

File: packages/scene-graph-mediator/client/src/exporter/ExportManager.ts [hunk 2]
```diff
@@ -54,16 +54,27 @@ export default class ExportManager {
   /**
    * Dynamically loads user defined plugin by absolute module path
    */
-  public loadPlugins(paths: string[]): void {
-    for (let i = 0; i < paths.length; i++) {
-      const Plugin = require(paths[i]).default;
-      const instance = new Plugin() as AssetExporterPlugin | SceneExporterPlugin;
+  public loadPlugins(plugins: string[] | AssetExporterPlugin[] | SceneExporterPlugin[]): void {
```
Comment (dolow) [inline packages/scene-graph-mediator/client/src/exporter/ExportManager.ts]: Preparation for accepting plugin instance.

**Comment by zprodev** [comment]
LGTM

**Comment by YoshioHara** [comment]
LGTM

**Pull request merged**